pkg/asset/installconfig/kubevirt/client.go

```diff
@@ -0,0 +1,240 @@
+/*
+Copyright 2018 The Kubernetes Authors.
```

Comment:
Why are these new files getting copyright years of 2018?

Reply:
Copied by mistake, removed